diff --git i/src/build/SharedDeps.zig w/src/build/SharedDeps.zig
index 4877686c4..9bf298133 100644
--- i/src/build/SharedDeps.zig
+++ w/src/build/SharedDeps.zig
@@ -251,7 +251,7 @@ pub fn add(
         );
 
         if (b.systemIntegrationOption("freetype", .{})) {
-            step.root_module.linkSystemLibrary("bzip2", dynamic_link_opts);
+            step.root_module.linkSystemLibrary("bz2", dynamic_link_opts);
             step.root_module.linkSystemLibrary("freetype2", dynamic_link_opts);
         } else {
             step.root_module.linkLibrary(freetype_dep.artifact("freetype"));
