diff --git i/pkg/harfbuzz/buffer.zig w/pkg/harfbuzz/buffer.zig
index b97c1bef4..dbb37887a 100644
--- i/pkg/harfbuzz/buffer.zig
+++ w/pkg/harfbuzz/buffer.zig
@@ -282,14 +282,6 @@
     /// the exact cluster values of each character, but is harder to use for
     /// clients, since clusters might appear in any order.
     characters = c.HB_BUFFER_CLUSTER_LEVEL_CHARACTERS,
-
-    /// In `graphemes`, non-base characters are merged into the cluster of the
-    /// base character that precedes them. This is similar to the Unicode
-    /// Grapheme Cluster algorithm, but it is not exactly the same. The output
-    /// is not forced to be monotone. This is useful for clients that want to
-    /// use HarfBuzz as a cheap implementation of the Unicode Grapheme Cluster
-    /// algorithm.
-    graphemes = c.HB_BUFFER_CLUSTER_LEVEL_GRAPHEMES,
 };

 /// The hb_glyph_info_t is the structure that holds information about the
